scst-store/multicluster-setup.hbs.md

```diff
@@ -45,69 +54,104 @@ metadata:
 ```
+## Copy AMR CloudEvent Handler edit token from the View cluster
 
-## Apply the CA certificate and authentication token to a new Kubernetes cluster
+Copy the AMR CloudEvent Handler token into an environment variable. You use
+this environment variable in the next step.
 
-Before you deploy the Build profile, you must apply the CA certificate and
-authentication token from the earlier steps. Then the Build profile deployment
-has access to these values.
+```bash
+CEH_EDIT_TOKEN=$(kubectl get secrets amr-cloudevent-handler-edit-token -n metadata-store -o jsonpath="{.data.token}" | base64 -d)
+```
+
+## Apply the SCST - Store CA certificates and SCST - Store tokens to Build cluster
+
+Once we have certificate and tokens copied, we apply the Metadata Store CA certificates, Metadata Store authentication token and CloudEvent Handler edit token to the Kubernetes cluster where you intend to install the Build profile.
 
-To apply the CA certificate and authentication token:
+### Apply the Metadata Store CA certificate and an authentication token to a new Kubernetes cluster
 
-1. With your kubectl targeted at the Build cluster, create a namespace for the CA
-certificate and authentication token.
+Before you deploy the Build profile, you must apply the Metadata Store CA certificate and
+Metadata Store authentication token from the earlier steps. Then the Build profile deployment
+will have access to these values.
+
+To apply the Metadata Store CA certificate and an authentication token:
+
+1. With your kubectl targeted at the Build cluster, create a namespace for the Metadata Store CA
+certificate and an authentication token.
 
     ```bash
     kubectl create ns metadata-store-secrets
     ```
 
-1. Apply the CA certificate `store_ca.yaml` secret YAML you generated earlier.
+1. Apply the Metadata Store CA certificate `store_ca.yaml` secret YAML you generated earlier.
 
     ```bash
     kubectl apply -f store_ca.yaml
     ```
 
-1. Create a secret to store the access token. This uses the `AUTH_TOKEN` environment variable.
+1. Create a secret to store the access token. This uses the Metadata Store `MDS_AUTH_TOKEN` environment variable.
 
     ```bash
     kubectl create secret generic store-auth-token \
-      --from-literal=auth_token=$AUTH_TOKEN -n metadata-store-secrets
+      --from-literal=auth_token=$MDS_AUTH_TOKEN -n metadata-store-secrets
     ```
 
-The cluster now has a CA certificate named  `store-ca-cert` and authentication
+The cluster now has a Metadata Store CA certificate named  `store-ca-cert` and a Metadata Store authentication
 token named `store-auth-token` in the namespace `metadata-store-secrets`.
 
+### Apply the CloudEvent Handler CA certificate and edit token to Build cluster
+
+Steps to apply the CloudEvent Handler CA certificate and edit token:
+
+1. Apply the CloudEvent Handler CA certificate `ceh_ca.yaml` secret YAML you generated earlier.
```

Comment:
Before you apply the cert and token, I believe you forgot the creation of the `amr-observer-system` namespace.

Reply:
This will be always there on the build cluster.

Reply:
But are we doing this before stuff is installed on the build cluster? Which means that NS will not be there?